dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -392,8 +393,7 @@ breeze release-management publish-docs \
     --package-filter 'apache-airflow-providers-*' \
     --override-versioned
 
-cd "${AIRFLOW_SITE_DIRECTORY}"
-breeze release-management add-back-references --airflow-site-directory --gen-type providers
+breeze release-management add-back-references --airflow-site-directory DIRECTORY --gen-type providers
```

Comment:
Do you mean to run this command we need to manually replace the DIRECTORY key word?

Reply:
Yeah that was the initial idea. But that makes the release management team's life inconvenient. Improvising here